### DISCUSSION TITLE
Additional cases of supporting Mica for everyone in Firefox theme

### DISCUSSION THREAD

**Discussion by black7375** [MicaForEveryone/MicaForEveryone#353] [Show and tell]
Mika support was entered at [v8.0.0](https://github.com/black7375/Firefox-UI-Fix/releases/tag/v8.0.0) of my theme.
You can add the following settings as an activation method.
- `userChrome.theme.transparent.frame` to `true`

I want to add it to the wiki, but I don't have permission.
https://github.com/MicaForEveryone/MicaForEveryone/wiki/FAQ

**Comment by Skyyblaze** [comment]
I'm curious, will this still work with Firefox 117? I checked your repo but couldn't find anything conclusive.

**Reply by black7375** [reply]
Oh, that's right. There's one condition due to the recent update.

Windows 7 and 8 support codes have been deleted from 117 and will only work if you use 115 ESR. (This content should be added)
- https://github.com/black7375/Firefox-UI-Fix/issues/744
- https://bugzilla.mozilla.org/show_bug.cgi?id=1850979

Instead, there is an official discussion
- https://bugzilla.mozilla.org/show_bug.cgi?id=1764822

**Reply by Skyyblaze** [reply]
Thanks for clarifying, seems I made the right call for staying on the ESR version for now.

**Comment by dongle-the-gadget** [comment]
I've decided to just change the FAQ page to say that MFE will work with Firefox 115 (or earlier) and any CSS styles with `-moz-appearance: -moz-win-glass` instead.

**Reply by black7375** [reply]
Make it sense!!

**Reply by dongle-the-gadget** [reply]
> Make it sense!!

What do you mean?

**Reply by black7375** [reply]
This means that the current wiki page looks good.

**Reply by dongle-the-gadget** [reply]
Oh, right.
